src/lib/DiscordWebhook/sendTradeSummary.ts

```diff
@@ -108,137 +112,126 @@ export default function sendTradeSummary(
+                                : '') +
+                            (AdditionalNotes
+                                ? (isShowKeyRate || isShowPureStock || isShowInventory ? '\n' : '') + AdditionalNotes
+                                : `\n[View my backpack](https://backpack.tf/profiles/${botInfo.steamID})`)
                     }
+                ],
+                footer: {
+                    text: `#${offer.id} • ${offer.partner.toString()} • ${time} • v${process.env.BOT_VERSION}`
                 }
-            ]
-        };
-
-        if (itemList === '-') {
-            acceptedTradeSummary.embeds[0].fields.shift();
-        } else if (itemList.length >= 1024) {
-            // first get __Status__ element
-            const statusElement = acceptedTradeSummary.embeds[0].fields.pop();
-
-            // now remove __Item list__, so now it should be empty
-            acceptedTradeSummary.embeds[0].fields.length = 0;
-
-            const separate = itemList.split('@');
-
-            let newSentences = '';
-            let j = 1;
-            separate.forEach((sentence, i) => {
-                if ((newSentences.length >= 800 || i === separate.length - 1) && !(j > 4)) {
-                    acceptedTradeSummary.embeds[0].fields.push({
-                        name: `__Item list ${j}__`,
-                        value: newSentences.replace(/@/g, '')
-                    });
-
-                    if (i === separate.length - 1 || j > 4) {
-                        acceptedTradeSummary.embeds[0].fields.push(statusElement);
-                    }
+            }
+        ]
+    };
 
-                    newSentences = '';
-                    j++;
-                } else {
-                    newSentences += sentence;
-                }
-            });
-        }
-
-        url.forEach((link, i) => {
-            sendWebhook(link, acceptedTradeSummary, 'trade-summary', i)
-                .then(() => {
-                    log.debug(`✅ Sent summary (#${offer.id}) to Discord${url.length > 1 ? `(${i + 1})` : ''}.`);
-                })
-                .catch(err => {
-                    log.debug(
-                        `❌ Failed to send trade-summary webhook (#${offer.id}) to Discord ${
-                            url.length > 1 ? ` (${i + 1})` : ''
-                        }: `,
-                        err
-                    );
+    if (combineList === '-') {
+        acceptedTradeSummary.embeds[0].fields.shift();
+    } else if (combineList.length >= 1024) {
+        // first get __Status__ element
+        const statusElement = acceptedTradeSummary.embeds[0].fields.pop();
+
+        // now remove __Item list__, so now it should be empty
+        acceptedTradeSummary.embeds[0].fields.length = 0;
+
+        const separate = itemList.split('@');
+
+        let newSentences = bot.options.manualReview.showItemPrices ? `${prices}\n\n` : '';
```

Comment:
This newSentences + line 207 below seem to stoke the failed webhook "embeds [0]" bug (https://discord.com/channels/664971400678998016/666909760666468377/793894735457812520).

If newSentenses is empty string, then __Item list__ can end up as an empty string which causes the discord error. (It doesn't like __Item list__ to be an empty string.

Reply:
https://github.com/TF2Autobot/tf2autobot/blob/964e5cd6f1d3148be78d35ce63bf0a84ef71ca72/src/lib/DiscordWebhook/sendTradeSummary.ts#L190-L191

Item list element will be removed if empty (nothing to show).

Reply:
https://gyazo.com/c87960a79e9afa91dd4fd7771221a7c1

Reply:
but will have a deep analysis on this later.

Reply:
👍 I might have some more detailed logs of the inputs soon

Reply:
So hit the case where:
```
debug: sendTradeSummary itemList: -
debug: sendTradeSummary combineList:
```
set by:
https://github.com/TF2Autobot/tf2autobot/blob/964e5cd6f1d3148be78d35ce63bf0a84ef71ca72/src/lib/DiscordWebhook/sendTradeSummary.ts#L50


then when the code reaches:
https://github.com/TF2Autobot/tf2autobot/blob/bd062f53f9cf71cfece0d1a55989ef38cdc05c7e/src/lib/DiscordWebhook/sendTradeSummary.ts#L190-L192
combineList is '' and length === 0 so no __Item list__ trimming will occur.

Reply:
Oh right. Maybe just need to add `|| combineList === ''` or `|| combineList.length === 0` here
https://github.com/TF2Autobot/tf2autobot/blob/964e5cd6f1d3148be78d35ce63bf0a84ef71ca72/src/lib/DiscordWebhook/sendTradeSummary.ts#L190

Reply:
That would probably work. Otherwise the combined list default should be "-" instead of empty string back in https://github.com/TF2Autobot/tf2autobot/blob/964e5cd6f1d3148be78d35ce63bf0a84ef71ca72/src/lib/DiscordWebhook/sendTradeSummary.ts#L48:L50

Reply:
Alright. Thanks for the help. I will push a commit soon.

Reply:
This looks good now. thanks